pkg/cmd/cli/cmd/request_project.go

```diff
@@ -113,23 +116,36 @@ func (o *NewProjectOptions) Run() error {
 		return err
 	}
 
-	if o.ProjectOptions != nil {
-		o.ProjectOptions.ProjectName = project.Name
-		o.ProjectOptions.ProjectOnly = true
-		o.ProjectOptions.SkipAccessValidation = true
+	if !o.SkipConfigWrite {
+		if o.ProjectOptions != nil {
```

Comment:
@deads2k Done. I still have to take a `clientConfig` to get the host, but definitely looks better.